alaoweb/2020.alaoweb.org
# Session sponsor logos#90

* adds sponsor info to session modal, poster modal, and poster layout
* addresses #90
* more changes will come when we get more kinds of sponsors 

###How to test:
* **Sessions**
  * go to the schedule page, click on a session. Should see Gold sponsors only at the bottom of the modal for non-keynote items. On keynotes, you should also see the keynote sponsors along with the Golds.
* **Posters**: 
  * go back one commit to (`git checkout 32526e5`) -- that will activate Derek's fake poster and turn posters on 
  * **you will need to do `bundle exec jekyll build && bundle exec jekyll serve`** to build the posters page
  * go to the session-search page; click on a poster to see Gold and Poster sponsors at the bottom of the modal
  * go to an individual poster page from the "posters" page"; the sponsors are on the right, except for small screens where they'll be at the bottom
